app/Http/Controllers/EpisodeManager.php

```diff
@@ -23,9 +33,21 @@ class EpisodeManager extends Controller
     */
     public function index()
     {
-        $episodes = Episode::all();
-
-        return view('dashboard/pages/view_episodes', compact('episodes'));
+        $data = [
+            "user"      =>  [  "total"     => $this->userRepository->getAllUser(),
+                                "online"    => $this->userRepository->getOnlineUsers()->count(),
+                                "offline"   => $this->userRepository->getOfflineUsers()->count() 
```

Comment:
**Code Quality - Nitpick**

"total" => $this->userRepository->getAllUser should start from the next line